coss/opensource_clubs/templates/opensource_clubs/entity_detail_page.html

```diff
@@ -1,109 +1,124 @@
 {% extends "opensource_clubs/category_base.html" %}
-{% load static wagtailcore_tags wagtailimages_tags %}
+{% load static wagtailcore_tags wagtailimages_tags gravatar %}
+
+{% block extra_css %}
+  <link rel="stylesheet" type="text/scss" href="{% static 'css/clubs.scss' %}">
+  <link rel="stylesheet" type="text/scss" href="{% static 'css/about.scss' %}">
+  <link rel="stylesheet" type="text/scss" href="{% static 'css/entity.scss' %}">
+{% endblock %}
 
 {% block header_class %}white-header{% endblock %}
 
 {% block jumbotron %}
-  <div class="container jumbotron club">
-    <div class="breadcrumbs">
-      <a href="{{ page.get_parent.url }}">
-        <img class="ml-2" src="{% static 'img/icons/left-arrow-small.svg' %}" alt="back">
-        <span>Open Source Clubs</span>
-      </a>
+  {% if page.get_first_image %}
```

Comment:
The two images in the entity page are the same (the one in the header and the one in the body). Shouldn't those two be different?

Reply:
We don't use `get_first_image` on the body anymore, since we have the Image gallery.

Reply:
Shouldn't this be editable through the admin?